src/App.tsx

```diff
@@ -1,98 +1,16 @@
 /* eslint-disable max-len */
+// import { title } from 'process';
 import React from 'react';
 
 import './App.scss';
-// import moviesFromServer from './api/movies.json';
+import moviesFromServer from './api/movies.json';
+import { MoviesList } from './components/MoviesList/MoviesList';
 
 export const App: React.FC = () => (
   <div className="page">
     <div className="page-content">
       <div className="movies">
```

Comment:
This div should be inside the MoviesList component also. Your page should look like this
<img width="1680" alt="image" src="https://user-images.githubusercontent.com/99403198/213685590-f5d94be0-d54d-4bc1-a748-91a0bde0d684.png">

Reply:
it was there already, I just missed it to delete in APP file) so one div was duplicated in 2 files